components/channel_header/channel_header.jsx

```diff
@@ -111,9 +111,7 @@ export default class ChannelHeader extends React.Component {
         }
     }
 
-    toggleFavorite = (e) => {
-        e.preventDefault();
```

Comment:
This is going to be really nice to get rid of all these preventDefault calls all over the place